### DISCUSSION TITLE
Stylus Input Latency on Linux

### DISCUSSION THREAD

**Discussion by imjustanotherdude** [flxzt/rnote#729] [Q&A]
Hello everyone:) 

I just installed Ubuntu 22.10 on my Surface Pro7 (thanks to the guys from Linux Surface) and i only use it for Handwriting.
On Windows its pretty decent, but I prefer Linux security wise. 
I installed several Handwriting Apps, but Rnote is by fare the most useful on. 
So thank you so much for that!!

The Problem is, that there is a little input latency (only on Linux)... which makes writing annoying.
Is there a way to improve that with a prediction algorithm or does anyone has a idea?
Thanks in advance for every little hint or information:)

**Comment by PhilDevProg** [comment]
You should read this comment from one of the linux-surface developers: https://github.com/linux-surface/linux-surface/discussions/593#discussioncomment-6307232
It explains why the latency is more on Linux than on Windows.
